mysql-test/t/blackhole-bug53588.test

```diff
@@ -0,0 +1,15 @@
+#
```

Comment:
Either just bug53588.test, or just extend the existing main.blackhole test (make sure to use an --echo # header block like other bugs in that file and put the snippet above the "end of 5.5 tests" marker)